core/src/main/java/org/eclipse/hono/util/CredentialsResult.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+
+package org.eclipse.hono.util;
+
+import io.vertx.core.json.JsonObject;
+
+/**
+ * A container for the result returned by Hono's credentials API.
+ *
+ */
+public final class CredentialsResult {
```

Comment:
FACK - I wanted to postpone the refactoring to directly after having the get operation agreed and merged.
There are plenty lines that will temporarily be doubled, but to make a separated refactoring pull request should make our life much easier, since it is focussed on just that.
Would do that before more credentials operations are implemented, WDYT?